### PR TITLE
added default civic button and fix issues

### PR DESCRIPTION
**Pull request by BouweCeunen**

### CODE REVIEW

File: src/react/components/CivicHeaderSection.tsx [hunk 1]
```diff
@@ -1,34 +1,33 @@
 import { useAnchorWallet, useConnection } from "@solana/wallet-adapter-react";
-import React from "react";
-import { Badge, GatewayStatus, useGateway } from "@civic/solana-gateway-react";
+import React, { useEffect } from "react";
```
Comment (SushantChandla) [inline src/react/components/CivicHeaderSection.tsx]: 'useEffect' is defined but never used

File: src/styles/index.scss [hunk 2]
```diff
@@ -51,12 +51,20 @@ button,
...
 	font-weight: bold !important;
+	height: 30px !important;
+}
+
+.sc-gsDKAQ {
+	svg {
+		width: 18px !important; 
+		height: 18px !important; 
+		top: 3px !important; 
+	}
```
Comment (BouweCeunen) [inline src/styles/index.scss:62]: done to make the civic button the same height as ours, other solution is to increase our button size to the size of the identity button

File: src/react/components/CivicHeaderSection.tsx [hunk 3]
```diff
@@ -1,34 +1,32 @@
...
-					clusterName={config.clusterConfig.name}
-					gatekeeperNetwork={props.gatekeeperNetwork}
-					publicKey={wallet.publicKey}
-					connection={connection.connection}
-				/>}
-				{!gatewayToken && <CivicButton
-					className={props.className}
-					text={GatewayStatus[gatewayStatus]}
-				/>}
+				{props.gatekeeperNetwork && <div className={"navbar-button"}>
```
Comment (RuvendeGroote) [inline src/react/components/CivicHeaderSection.tsx]: last time we agreed to do little cleanups of the code and we agreed to use the 'header' terminology when talking about the buttons on top. maybe renaming this navbar-button class is a good little cleanup. stuff like that keeps the code healthy

File: src/styles/navbar.scss [hunk 4]
```diff
@@ -52,4 +52,5 @@
 img.logo {
 	max-height: 40px;
 	margin-left: 20px;
-}
\ No newline at end of file
+}
+
```
Comment (RuvenSalamon) [inline src/styles/navbar.scss:56]: remove empty newline and use prettier

File: src/react/components/CivicHeaderSection.tsx (hunk 3 above)
Comment (RuvendeGroote) [reply src/react/components/CivicHeaderSection.tsx]: and use prettier. configure your editor

**Review by RuvendeGroote — CHANGES_REQUESTED** [review]

File: src/react/components/CivicHeaderSection.tsx [hunk 5]
```diff
@@ -1,34 +1,32 @@
...
-					clusterName={config.clusterConfig.name}
-					gatekeeperNetwork={props.gatekeeperNetwork}
-					publicKey={wallet.publicKey}
-					connection={connection.connection}
-				/>}
-				{!gatewayToken && <CivicButton
-					className={props.className}
-					text={GatewayStatus[gatewayStatus]}
-				/>}
+				{props.gatekeeperNetwork && <div className={"header-button"}>
```
Comment (RuvendeGroote) [inline src/react/components/CivicHeaderSection.tsx]: prettier gives me different results here

**Review by RuvendeGroote — APPROVED** [review]

**Review by MaximPiessen — APPROVED** [review]

**Pull request merged**